l10n.yaml

```diff
@@ -1,3 +1,5 @@
 arb-dir: lib/src/localization
+output-dir: lib/src/localization/gen
 template-arb-file: app_en.arb
 output-localization-file: app_localizations.dart
+synthetic-package: false
```

Comment:
What was the purpose of this change?

Reply:
When I swapped the Window_manger for the forked version the localization files were not being generated anymore, so I did the manual steps. I included flutter_gen, changed it to generate   the files inside the output_dir with those two lines. 
I didn't quite like this unreliable behavior so I also created a "suggestion issues" to evaluate Slang as an i18n solution.